### PR TITLE
Bugfix, mixed up the order

### PR DESCRIPTION
**Pull request by jzwar**
# Multipatch bug

Mixed up the order in the connectivity template in the splinepy backend

### CODE REVIEW

**Review by clemens-fricke — APPROVED** [review]
Seems to be ok.

**Review by j042 — CHANGES_REQUESTED** [review]
I request a test

File: splinepy/multipatch.py [hunk 1]
```diff
@@ -116,6 +116,8 @@ def interfaces(self, con):
             raise ValueError(
                     "Connectivity must be stored in a numpy 2D array."
             )
+        if con.dtype is not np.int32:
+            con = con.astype(np.int32)
```
Comment (j042) [inline splinepy/multipatch.py:120]:
```suggestion
        con = utils.data.enforce_contiguous(con, np.int32)
```

File: splinepy/utils/data.py [hunk 2]
```diff
@@ -218,14 +218,12 @@ def enforce_contiguous(array, dtype=None):
     -------
     contiguous_array: array-like
     """
-    if isinstance(array, np.ndarray):
+    if issubclass(type(array), np.ndarray):
```
Comment (j042) [inline splinepy/utils/data.py:221]: I'd keep is instance

File: splinepy/utils/data.py [hunk 3]
```diff
@@ -218,14 +218,12 @@ def enforce_contiguous(array, dtype=None):
...
         if array.flags["C_CONTIGUOUS"]:
-            return array
-        else:
-            return np.ascontiguousarray(array, dtype=dtype)
+            if array.dtype is dtype:
+                return array
 
-    else:
-        return array
+    return np.ascontiguousarray(array, dtype=dtype)
```
Comment (j042) [inline splinepy/utils/data.py:226]: what are you gonna do is dtype is none?

**Review by j042 — CHANGES_REQUESTED** [review]

File: splinepy/utils/data.py (hunk 3 above)
Comment (j042) [reply splinepy/utils/data.py:226]: check if c contiguous and dtype is None, you return, else return np.ascontiguousarray(array, dtype=dtype). If array is contiguous and dtype is the same, you will just get the same array

File: splinepy/utils/data.py (hunk 2 above)
Comment (jzwar) [reply splinepy/utils/data.py:221]: OK, I just remembered a conversation we had, I wasn't sure about the outcome anymore. Changed it back

Comment (j042) [reply splinepy/utils/data.py:221]: thank you!

**Comment by j042** [comment]
I think we have different format setup

**Review by j042 — APPROVED** [review]
thanks!

**Pull request merged**